build_catalog.sh

```diff
@@ -43,9 +49,9 @@ fi
 # Build/push the new bundle
 pushd deploy/bundle
   if [[ $prev_version != "" ]]; then
-    prev="REPLACE_VERSION=$prev_version"
+    prev="export REPLACE_VERSION=$prev_version"
   fi
-  IMAGE=$BUNDLE_IMAGE_NAME IMAGE_TAG=$current_commit VERSION=$version $prev make bundle
+  (export IMAGE=$BUNDLE_IMAGE_NAME && export IMAGE_TAG=$current_commit && export VERSION=$version && $prev; make bundle)
```

Comment:
This looks so weird... You shouldn't need to to export if you define the vars in the command... maybe what bash doesn't like is how you were defining a var in a string that you interpolate (`prev="REPLACE_VERSION=$prev_version"`). I guess it would be clearer if you exported the vars separately and you exported `REPLACE_VERSION` inside the `if [[ $prev_version != "" ]]` conditional

Reply:
I don't really like the idea of exporting variables into the env that are only needed locally.  Bash didn't like $prev_version interpolation, it kept treating it as a command no matter what I tried.  I ended with this solution so as to not pollute the execution of the env rather than doing straight exports.

Reply:
But you're already exporting variables unless those parenthesis do some magic I don't understand

The other solution would be then to execute different commands in the previous `if` unless you can live with `REPLACE_VERSION` set to `""`